mora/service/employee.py

```diff
@@ -598,31 +590,22 @@ def terminate_employee(employee_uuid):
 
     c = lora.Connector(effective_date=date)
 
-    engagements = c.organisationfunktion.get_all(
-        tilknyttedebrugere=employee_uuid,
-        funktionsnavn=keys.ENGAGEMENT_KEY)
-    for engagement in engagements:
-        engagement_uuid = engagement[0]
-        terminate_engagement(engagement_uuid, date)
-
-    associations = c.organisationfunktion.get_all(
-        tilknyttedebrugere=employee_uuid,
-        funktionsnavn=keys.ASSOCIATION_KEY)
-    for association in associations:
-        association_uuid = association[0]
-        terminate_association(association_uuid, date)
-
-    # TODO: Terminate IT
-    # TODO: Terminate Kontakt
-    # TODO: Terminate Rolle
-    # TODO: Terminate Leder
-
-    leaves = c.organisationfunktion.get_all(
-        tilknyttedebrugere=employee_uuid,
-        funktionsnavn=keys.LEAVE_KEY)
-    for leave_obj in leaves:
-        leave_uuid = leave_obj[0]
-        leave.terminate_leave(leave_uuid, date)
+    # Org funks
+    types = [
+        (keys.ENGAGEMENT_KEY, 'Afslut engagement'),
+        (keys.ASSOCIATION_KEY, 'Afslut tilknytning'),
+        (keys.ROLE_KEY, 'Afslut rolle'),
+        (keys.LEAVE_KEY, 'Afslut orlov'),
+    ]
+    for funk_type in types:
```

Comment:
You should unwrap this in one line, e.g.:

```python
for key, note in types:
```

Also, I'd suggest using one note for all actions — after all, they are infact separate results of the same user intent, and perhaps tuple since you don't need mutability.